lnbits/extensions/gerty/number_prefixer.py

```diff
@@ -22,36 +23,29 @@ def si_classifier(val):
         -24: {"long_suffix": "yocto", "short_suffix": "y", "scalar": 10**-24},
     }
     exponent = int(math.floor(math.log10(abs(val)) / 3.0) * 3)
-    return suffixes.get(exponent, None)
+    suffix = suffixes.get(exponent)
+    assert suffix, f"could not classify: {val}"
+    return suffix
```

Comment:
> **Warning** now it throws instead of returning `None`

Reply:
i think that is actually accurate? @blackcoffeexbt

Reply:
@blackcoffeexbt tested it